src/views/ContinuationIn/ContinuationInBusinessBc.vue

```diff
@@ -6,7 +6,8 @@
         <h2>Company Name</h2>
       </header>
 
-      <div>** Correct Name component goes here **</div>
+      <NameRequestInfo />
+      <NameTranslations />
```

Comment:
These need to be in a v-card so that they match the Figma.

Figma:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/0ad5c34e-2f99-4000-9129-8eb5a1e74b79)

Actual:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/bd4a1bfd-dfa2-4fb8-9198-66ff4df22e06)

Reply:
Thank you. Updated:
![image](https://github.com/bcgov/business-create-ui/assets/122495122/865a093a-3f00-422d-8c1f-e34a35eadd78)

Reply:
Yes, I tested with a named company:
![image](https://github.com/bcgov/business-create-ui/assets/122495122/7ea32f34-3e3b-41ac-abcd-0902abf5e0f4)

Reply:
I'm wondering if you feel the 2 spaces should be smaller?
![image](https://github.com/bcgov/business-create-ui/assets/122495122/7d8b4139-032a-4e47-9d31-fc4fbb2266f3)
@severinbeauvais 
Thank you!

Reply:
See how the other filings wrap the components above and you can probably copy it.

Reply:
The v-card looks correct but, yes, I think the top and bottom whitespace should be smaller. Compare with other v-cards.

The named continuation in application looks good 👍

Reply:
Thank you Severin. I updated the top and bottom whitespaces:
![image](https://github.com/bcgov/business-create-ui/assets/122495122/d3fc36d8-4d88-4b4a-9090-e8dc0b1d0bf3)